scripts/yarn-list.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python
+"""A utility that outputs package information _like_ `pip list ...`, but for
+yarn packages.
+
+TODO:
+- dynamically calculate column widths rather than using hard-coded widths
```

Comment:
before you invest in any table utils you should look at `corehq.util.markup.SimpleTableWriter`. It also doesn't do automatic table widths but it probably could